thirdparty/misc/triangulator.h

```diff
@@ -21,9 +21,9 @@
 #ifndef TRIANGULATOR_H
 #define TRIANGULATOR_H
 
-#include "math_2d.h"
 #include "list.h"
 #include "set.h"
+#include "vector2.h"
```

Comment:
You could include "vector2.h" as well as other divided classes into "math_2d.h".. and math_2d.h could be collection of "include". And no need to change this in such case..

Reply:
However this approach could improve readability.. its your choise, I just mentioned about this possibility

Reply:
That is how it currently is, `math_2d.h` has an include for `vector2.h`. But if you're not using the rest of `math_2d.h` it would be better organized to only include what you need, yes?